app/models/settings/base.rb

```diff
@@ -127,6 +136,8 @@ def convert_string_to_value_type(type, value, separator: nil)
           value.to_f
         when :big_decimal
           value.to_d
+        when :markdown
+          ContentRenderer.new(value).process&.processed_html
```

Comment:
`ContentRenderer#process` should always return `Result` or raise an exception, so I would remove `&.` - let's be more confident.